### ISSUE TITLE
[observability] update GitpodWorkspaceNodeHighNormalizedLoadAverage to trigger sooner

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/gitpod#7552]
## Is your feature request related to a problem? Please describe

[Our observability is described here.](https://github.com/gitpod-io/gitpod/tree/main/operations/observability/mixins)

[We have existing alerts for node storage.](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/cross-teams/rules/components/nodes/alerts.libsonnet#L13)

[We have an existing warning](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L13), which indicates when nodes may be getting over whelmed, but it doesn't fire early enough. For example, this would be a signal that node resources (CPU and memory) are taxed, potentially degrading the experience for users in workspaces.

The current warning will not fire unless the condition has been met for [10 minutes](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L17), where the [normalized load is > 10](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L23).

## Describe the behaviour you'd like

The warning should fire when our nodes are under significant load. What we've seen, is that it's current params prevent firing even when load is significant.

A `for` of `5m` seems better than `10m`.
A `expression` > 5, seems better than 10.

In other words, if we cut what is currently there in half, and look at the data over the last 7 days, it would have triggered warnings (not errors) a few times. This would of given us a signal that a cluster was taxed, and the opportunity to investigate "the current state".

Inspect the load for the EU and US clusters for the last seven days. What do you think?

### How to test
We'll need to test using `loadgen`, I'm certain @aledbf knows how to do this, and would be willing to make a short Loom to help us use it. ;)

### Other
This is a warning, there is no runbook, which is okay, because it does not page.

Why do this? We encountered issues recently where load was high, but this alert did not fire.

**Comment by ArthurSens** [comment]
Hey Kyle, we already have an alert for this(See [related PR](https://github.com/gitpod-io/gitpod/pull/6241#issuecomment-944357589)) but it is intentionally not set to critical due to its noise (e.g [alert](https://gitpod.slack.com/archives/C01TNS9FCQ3/p1641513409002900)). It is not exactly CPU usage, but normalized load average(current load average divided by the number of CPU cores of the node), which is a little bit less noisy.


It is intentionally not set as critical because of a couple of things:

* CPU spikes are common and often resolves by themselves, should potentially wake up someone in the middle of the night because of a sudden spike? 
* Does High CPU usages reflect a bad user experience? Most of the time, it does not. We really encourage building metrics that represent user experience more accurately to build alerts on. See related [blog post](https://www.gitpod.io/blog/journey-to-site-reliability-engineering-at-gitpod)

When building alerts used for on-call, we should really focus on Symptom-based alerts instead of possible causes 🙂

**Comment by kylos101** [comment]
@ArthurSens I updated the description of this issue to address the spirit of your feedback. Thank you!

**Comment by kylos101** [comment]
@aledbf what do you think of these params? 

A for of 5m seems better than 10m.
A expression triggers on 5, rather htan 10.

Need to be more aggressive? I cut the current ones in half, we could keep cutting until it is "right".

**Comment by aledbf** [comment]
> A expression triggers on 5, rather htan 10.

I think we need a signal now. If this is too aggressive we can adjust it later.
